po/sv.po

```diff
@@ -2025,40 +2010,39 @@ msgstr ""
 #. second %s is the tail itself.
 #: rust/libnewsboat/src/filterparser.rs:426
 msgid "Parse error: trailing characters after position %s: %s"
-msgstr ""
+msgstr "Tolkningsfel: släpande tecken efter position %s: %s"
 
 #. The first %s is a zero-based offset into the string, the second %s is the
 #. description of what the program expected at that point.
 #: rust/libnewsboat/src/filterparser.rs:433
 msgid "Parse error at position %s: expected %s"
-msgstr ""
+msgstr "Tolkningsfel vid position %s: förväntade %s"
 
 #: rust/libnewsboat/src/filterparser.rs:437
 msgid "Internal parse error"
-msgstr ""
+msgstr "Internt tolkningsfel"
 
 #: rust/libnewsboat/src/fslock.rs:57
-#, fuzzy
 msgid "Failed to open lock file '%s': %s"
-msgstr "Fel vid inläsning av webbflöde `%s': %s"
+msgstr "Kunde inte öppna låsfil`%s': %s"
 
 #: rust/libnewsboat/src/fslock.rs:60 rust/libnewsboat/src/fslock.rs:80
 #: rust/libnewsboat/src/fslock.rs:112
 msgid "<filename containing invalid UTF-8 codepoint>"
-msgstr ""
+msgstr "<filnamn innehållande ogiltig UTF-8-kodpunkt>"
 
 #: rust/libnewsboat/src/fslock.rs:77
 msgid "Failed to write PID to lock file '%s': %s"
-msgstr ""
+msgstr "Misslyckades att skriva PID till låsfil: '%s': %s"
```

Comment:
Is this colon really needed? (It might be — I don't know Swedish.)

```suggestion
msgstr "Misslyckades att skriva PID till låsfil '%s': %s"
```

Reply:
No, it's not needed - thanks!